crates/iota-indexer/tests/rpc-tests/read_api.rs

```diff
@@ -1234,18 +1239,83 @@ fn try_get_past_object() {
         runtime,
         store,
         client,
-        ..
+        cluster,
     } = ApiTestSetup::get_or_init();
+
     runtime.block_on(async move {
         indexer_wait_for_checkpoint(store, 1).await;
 
+        let object_id = ObjectID::random();
+        let version = SequenceNumber::new();
+
+        // Test: Object does not exist
         let result = client
-            .try_get_past_object(ObjectID::random(), SequenceNumber::new(), None)
+            .try_get_past_object(object_id, version, None)
+            .await
+            .expect("RPC call should succeed");
+
+        assert!(
```

Comment:
Could we use `assert_eq` here?

We know the exact expected value: `IotaPastObjectResponse::ObjectNotExists(object_id)`